cmd/lotus-shed/chain_index.go

```diff
@@ -0,0 +1,191 @@
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"time"
+
+	"github.com/urfave/cli/v2"
+	"golang.org/x/xerrors"
+
+	"github.com/filecoin-project/go-state-types/abi"
+
+	lcli "github.com/filecoin-project/lotus/cli"
+)
+
+var chainIndexCmds = &cli.Command{
+	Name:            "chainindex",
+	Usage:           "Commands related to managing the chainindex",
+	HideHelpCommand: true,
+	Subcommands: []*cli.Command{
+		withCategory("chainindex", validateBackfillChainIndexCmd),
+	},
+}
+
+var validateBackfillChainIndexCmd = &cli.Command{
+	Name:  "validate-backfill",
```

Comment:
just "validate" might be enough for this I reckon, adding `--backfill` makes it kind of obvious what it's doing

Reply:
I do want to retain this name just to make it VERY obvious it can do both so users dont trip :)